homeassistant/components/device_tracker/owntracks.py

```diff
@@ -22,20 +23,28 @@
 
 LOCATION_TOPIC = 'owntracks/+/+'
 EVENT_TOPIC = 'owntracks/+/+/event'
+WAYPOINT_TOPIC = 'owntracks/{}/{}/waypoint'
```

Comment:
Perhaps too fussy, so feel free to ignore - but you always set the second `{}` to `+` - so you could hard code here.